advocacy_docs/edb-postgres-ai/ai-accelerator/retrievers/usage.mdx

```diff
@@ -1,17 +1,17 @@
 ---
-title: "Pipelines - Retrievers - Usage"
+title: "Pipelines retrievers usage"
 navTitle: "Usage"
 deepToC: true
 description: "Usage of retrievers in AI Accelerator Pipelines."
 ---
 
 ## Creating a new retriever configuration
 
-There are two types of retrievers: table and volume. Given the different nature of the data sources, and the options required for each we have different functions to create them.
+There are two types of retrievers: table and volume. Given the different nature of the data sources and the options required for each, there are different functions to create them.
 
 ## Retriever for a table data source
 
-The [aidb.create_retriever_for_table](../reference/retrievers#aidbcreate_retriever_for_table) function is used to create a retriever for a table data source. This is the function signature, you can see many of those are optional and have defaults.
+The [aidb.create_retriever_for_table](../reference/retrievers#aidbcreate_retriever_for_table) function is used to create a retriever for a table data source. This is the function signature. You can see many of those are optional and have defaults.
```

Comment:
This might be a vocabulary I'm not familiar with. I found the word "those" confusing here. Does this mean:

You can see many of the function's parameters are optional and have defaults.

Wasn't sure if signature had a different vocabulary and doesn't use the word parameters.